### DISCUSSION TITLE
Allow controlling volume directly in media-control-card

### DISCUSSION THREAD

**Discussion by JeffersonBledsoe** [home-assistant/frontend#11905] [Dashboard]
To change the volume currently, you have to open the more info dialogue. It would be nice to be able to control the volume directly from the card. The card config could have an option to show/ hide the volume control. There could be the possibility of having multiple ways of displaying the volume control in future PRs (such as always showing the full volume bar visible like in the Spotify app, or having the volume bar appear only when clicked on such as on YouTube) which could also be configurable through the card's edit interface to accommodate user preference.

**Comment by krazos** [comment]
Volume control would be a great addition to the Media Control card.  In the interest of simplicity, I would be inclined to implement it in the way that it was recently added to the media player bar (i.e., a small button to the left of the "browse media" button that toggles the volume slider). 

![Screenshot from 2022-03-08 11-05-11](https://user-images.githubusercontent.com/18428022/157277045-97e449d0-b3ac-44f6-9968-ef6db7dad9e3.png)

**Reply by FHFS** [reply]
Maybe use a custom card?
![image](https://user-images.githubusercontent.com/9058444/183266690-d5dd5f94-edbe-4b9e-a71c-176872d279f5.png)
```
  - type: custom:mini-media-player
    entity: media_player.squeezelite_fcdad4
  - type: media-control
    entity: media_player.squeezelite_fcdad4
```

**Comment by krazos** [comment]
Bumping this feature request.  There are of course custom card alternatives, but it would be nice for the stock media player card to have more intuitive volume control.  I have attached a mock-up showing one potential approach -- the inclusion of a small volume icon, which when clicked could pop up a volume slider.  This approach likely could reuse some code from the media browser, which takes a similar approach to volume control.

![Media Player Card Mock-up with Volume Button](https://user-images.githubusercontent.com/18428022/189365862-fb37f027-d76a-4612-aae7-3433272ab02e.png)
